src/meshes/dgmulti_meshes.jl

```diff
@@ -43,52 +39,67 @@ function Base.show(io::IO, ::MIME"text/plain", mesh::VertexMappedMesh{NDIMS, Ele
 end
 
 """
-    VertexMappedMesh(vertex_coordinates::NTuple{NDIMS, Vector{Tv}}, EToV, rd::RefElemData;
-                     is_on_boundary = nothing,
-                     is_periodic::NTuple{NDIMS, Bool} = ntuple(_->false, NDIMS)) where {NDIMS, Tv}
+  DGMultiMesh(vertex_coordinates::NTuple{NDIMS, Vector{Tv}}, EToV, rd::RefElemData;
+              is_on_boundary = nothing,
+              periodicity::NTuple{NDIMS, Bool} = ntuple(_->false, NDIMS)) where {NDIMS, Tv}
 
 - `vertex_coordinates` is a tuple of vectors containing x,y,... components of the vertex coordinates
 - `EToV` is a 2D array containing element-to-vertex connectivities for each element
 - `rd` is a `RefElemData` from `StartUpDG.jl`, and contains information associated with to the
   reference element (e.g., quadrature, basis evaluation, differentiation, etc).
 - `is_on_boundary` specifies boundary using a `Dict{Symbol, <:Function}`
-- `is_periodic` is a tuple of booleans specifying periodicity = `true`/`false` in the (x,y,z) direction.
+- `periodicity` is a tuple of booleans specifying if the domain is periodic `true`/`false` in the
+   (x,y,z) direction.
 """
-function VertexMappedMesh(vertex_coordinates::NTuple{NDIMS, Vector{Tv}}, EToV::Array{Ti,2}, rd::RefElemData;
-                          is_on_boundary = nothing,
-                          is_periodic::NTuple{NDIMS, Bool} = ntuple(_->false, NDIMS)) where {NDIMS, Tv, Ti}
+function DGMultiMesh(vertex_coordinates::NTuple{NDIMS, Vector{Tv}}, EToV::Array{Ti,2}, rd::RefElemData;
+                     is_on_boundary = nothing,
+                     periodicity=ntuple(_->false, NDIMS), kwargs...) where {NDIMS, Tv, Ti}
+
+  if haskey(kwargs, :is_periodic)
+    # TODO: DGMulti. Deprecate `is_periodic` in version 0.5
```

Comment:
```suggestion
    # TODO: DGMulti, v0.5. Remove deprecated keyword
```
To unify the TODO notes related to deprecations